api/src/server/gql/hardware_spec.rs

```diff
@@ -139,11 +139,11 @@ impl HardwareSpecConnection {
 
     fn get_total_count(&self, context: &Context) -> ServerResult<i32> {
         match hardware_specs::table
-            .select(dsl::count(dsl::count_star()))
+            .select(dsl::count_star())
             .get_result::<i64>(&context.get_db_conn()?)
         {
             // Convert i64 to i32 - if this fails, we're in a rough spot
-            Ok(count) => Ok(count.try_into()?),
+            Ok(count) => Ok(count.try_into().unwrap()),
```

Comment:
I changed these errors to `unwrap`s cause if they happen, it means the table is too big for its length to fit in an `i32`. I think we want a panic to occur in these cases, so that they get logged and we can potentially get notified about them. I'm planning on adding logic soon to catch panics in response handlers, so this won't crash the server.